src/components/Shared/LinePlot/LinePlotComponent.tsx

```diff
@@ -659,7 +659,7 @@ export class LinePlotComponent extends React.Component<LinePlotComponentProps> {
         }
 
         this.props.multiPlotPropsMap?.forEach((multiPlotProp, key) => {
-            if (key === "colormapScaling") { // TODO: remove this to fully support multiple lines
+            if (key === "colormapScaling" || multiPlotProp.noExport) { // TODO: remove (key === "colormapScaling") to fully support multiple lines
```

Comment:
@TienHao why can't we make these changes in this PR as well? Surely it would just be a couple of lines?

Reply:
I think "remove this" is for `this.props.multiPlotPropsMap` (maybe added by @duidae ?)

Reply:
I see what @veggiesaurus mean. I have added `noExport: true` to the MultiPlotProps of colormapScaling.